docs/en/advanced/Performance-Tuning.md

```diff
@@ -33,11 +33,11 @@ The following is a checklist to run through to address common problems when tuni
    the compute application is likely not interfacing with a local Alluxio worker.
    The Alluxio client uses hostname matching to discover a local Alluxio worker;
    check that the client and worker use the same hostname string.
-   Configuring `alluxio.locality.node` and `alluxio.worker.hostname` sets the client and worker
```

Comment:
> Configuring `alluxio.locality.node` and `alluxio.worker.hostname` sets the client and worker
This is very counterintuitive compared to 
> Configuring `alluxio.user.hostname` and `alluxio.worker.hostname` sets the client and worker

Reply:
Yes, but then why was `user.hostname` deprecated? I think it's fine because if users are configuring this they're probably more experienced/knowledgable in this realm anyways.

A possible stop-gap here is to make `alluxio.user.hostname` and alias of `alluxio.locality.node`